app/views/contexts/summary/question.py

```diff
@@ -125,14 +136,21 @@ def _concatenate_answers(self, answer_store, concatenation_type):
                 continue
 
             values_to_concatenate.extend(
-                answer_value if isinstance(answer_value, list) else [answer_value]
+                # Type ignore: This will be optional as we have answer_value check
+                answer_value
+                if isinstance(answer_value, list)
+                else [answer_value]  # type: ignore
             )
 
         return answer_separator.join(str(value) for value in values_to_concatenate)
 
     def _build_answer(
-        self, answer_store, question_schema, answer_schema, answer_value=None
-    ):
+        self,
+        answer_store: AnswerStore,
+        question_schema: QuestionSchemaType,
+        answer_schema: Mapping[str, Any],
+        answer_value: Optional[AnswerValueEscapedTypes] = None,
+    ) -> Any:
```

Comment:
This should be more explicit. This should be predictable?

Reply:
Sure.. I will update it.